client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -19,6 +19,10 @@ const useStyles = makeStyles((theme: Theme) => ({
   forgot: {
     paddingRight: 10,
   },
+  buttonsWrapper: {
+    display: 'flex',
```

Comment:
These styles can directly be added to the `Box` component. Ditto where applicable 

Eg: `<Box display="flex" alignItems="center".../>`